### PR TITLE
Update repo-config and setuptools

### PR DESCRIPTION
**Pull request by flora-hofmann-frequenz**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 [build-system]
 requires = [
-  "setuptools == 68.1.0",
-  "setuptools_scm[toml] == 7.1.0",
+  "setuptools == 75.5.0",
+  "setuptools_scm[toml] == 8.1.0",
   "frequenz-repo-config[lib] == 0.10.0",
```
Comment (cwasicki) [inline pyproject.toml]: what about this version?

**Review by cwasicki — COMMENTED** [review]
Isn't there anything else to update? IIUC there is a migration script

**Comment by flora-hofmann-frequenz** [comment]
> Isn't there anything else to update? IIUC there is a migration script

I've ran the migration script. Could you please review again?

File: pyproject.toml (hunk 1 above)
Comment (cwasicki) [reply pyproject.toml]: Doesn't this need updating?

Comment (flora-hofmann-frequenz) [reply pyproject.toml]: It is updating according to the repo config specs. `75.5.0` and `8.1.0` are the versions required.

Comment (flora-hofmann-frequenz) [reply pyproject.toml]: This is also in PR #119 and #120 from Dependabot.

Comment (cwasicki) [reply pyproject.toml]: I mean why is repo config not updated here?

Comment (flora-hofmann-frequenz) [reply pyproject.toml]: Sorry, I thought you meant the setup tools update... Will check!

**Review by cwasicki — APPROVED** [review]

**Pull request merged**